### PR TITLE
Add Chelsea Ho's "The Future of Product Personalisation"

### PR DESCRIPTION
**Pull request by typ3c4t**
[asana](https://app.asana.com/0/0/1200279623929003/f)

### CODE REVIEW

**Comment by typ3c4t** [comment]
the blog post looks good when built and run, but in the notion there was an embedded youtube video, which is now simply a hyperlink. thoughts on how to best render this?

**Comment by jhrcek** [comment]
On youtube you can get a piece of html which embeds the video into the page.
Open the video on youtube, click share, click "embed" and copy paste the html fragment directly into markdown.

Looks like this for me. If needed you can lookup google's youtube video embedding guide and tweak what features of embedded player are allows
```html
<iframe width="560" height="315" src="https://www.youtube.com/embed/X8IBgyAMgn4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

**Comment by typ3c4t** [comment]
nice :) I sized the video to match the width of the images; looks good!

File: blog/2021-08-24-The-Future-of-Product-Personalisation.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+> The **experiential personalisation** improves the overall customer experience, giving them a sense of agency and assurance that the product they receive at the end is truly the best fit for them.
+
+Experiential personalisation (in the product experience), in essence, takes on the cognitively-taxing parts of the decision-making process whilst still ensuring the customer feels included. 
+
+1. **Lumen** 
+
+<iframe width="895" height="503.4375" src="https://www.youtube.com/embed/X8IBgyAMgn4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (jhrcek) [inline blog/2021-08-24-The-Future-of-Product-Personalisation.md:48]: Before publishing this, I'd double-check with Eileen if it's ok for us to publish what seems to me like "marketing" material for another company's product.

**Review by jhrcek — APPROVED** [review]
LGTM.

**Pull request merged**